win32ss/reactx/ntddraw/dxeng.c

```diff
@@ -481,6 +481,8 @@ DxEngGetDCState(HDC hDC,
         DC_UnlockDc(pDC);
     }
 
+    DPRINT1("Return value %08lx\n", retVal);
```

Comment:
Is this still useful?

Reply:
Yes, because the return value from type `2` is still valid, and, besides that, this `DPRINT` displays the values from all 3 types, not only from the 2nd one.